tests/integration.rs

```diff
@@ -2713,7 +2713,7 @@ mod tests {
         });
     }
 
-    #[cfg_attr(not(feature = "mmio"), test)]
+    #[cfg_attr(feature = "mmio", test)]
```

Comment:
But this was not the point of this test, we wanted to make sure PCI could work without ACPI.